// @khanisak/temperature-converter/dist/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.units = exports.Romer = exports.Reamur = exports.Rankine = exports.Newton = exports.Kelvin = exports.Fahrenheit = exports.Delisle = exports.Celcius = void 0;
var Celcius_1 = require("./class/Celcius");
Object.defineProperty(exports, "Celcius", { enumerable: true, get: function () { return Celcius_1.Celcius; } });
var Delisle_1 = require("./class/Delisle");
Object.defineProperty(exports, "Delisle", { enumerable: true, get: function () { return Delisle_1.Delisle; } });
var Fahrenheit_1 = require("./class/Fahrenheit");
Object.defineProperty(exports, "Fahrenheit", { enumerable: true, get: function () { return Fahrenheit_1.Fahrenheit; } });
var Kelvin_1 = require("./class/Kelvin");
Object.defineProperty(exports, "Kelvin", { enumerable: true, get: function () { return Kelvin_1.Kelvin; } });
var Newton_1 = require("./class/Newton");
Object.defineProperty(exports, "Newton", { enumerable: true, get: function () { return Newton_1.Newton; } });
var Rankine_1 = require("./class/Rankine");
Object.defineProperty(exports, "Rankine", { enumerable: true, get: function () { return Rankine_1.Rankine; } });
var Reamur_1 = require("./class/Reamur");
Object.defineProperty(exports, "Reamur", { enumerable: true, get: function () { return Reamur_1.Reamur; } });
var Romer_1 = require("./class/Romer");
Object.defineProperty(exports, "Romer", { enumerable: true, get: function () { return Romer_1.Romer; } });
var units_1 = require("./units");
Object.defineProperty(exports, "units", { enumerable: true, get: function () { return units_1.units; } });
